src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c

```diff
@@ -164,8 +164,8 @@ static Gfx* sJabuDoorDLists[] = {
 };
 
 static void* D_809982D4[] = {
-    object_bdoor_Tex_0065C0, object_bdoor_Tex_0035C0, object_bdoor_Tex_0055C0, object_bdoor_Tex_0045C0,
-    object_bdoor_Tex_000000, object_bdoor_Tex_0025C0, object_bdoor_Tex_0015C0,
+    gBossDoorDefaultTex,      gBossDoorFireTex,   gBossDoorWaterTex,  gBossDoorShadowTex,
+    gBossDoorGanonsCastleTex, gBossDoorForestTex, gBossDoorSpiritTex,
```

Comment:
this could prob be named after boss door textures, but dragorn also probably does that in his door_shutter pr

Reply:
Yeah, he calls it `sBossDoorTextures`. I think I'll just leave it as-is and let his PR name it.